.travis.yml

```diff
@@ -55,15 +77,19 @@ script:
       docker tag "$IMG_TAG" "$BASE_PUSH_TARGET:$BUILD_STAMP"
       docker tag "$IMG_TAG" "$BASE_PUSH_TARGET:$BUILD_VERSION"
       docker tag "$IMG_TAG" "$BASE_PUSH_TARGET:$BUILD_VERSION-$BUILD_INFO"
+      docker tag "$IMG_TAG" "$BASE_PUSH_TARGET:$TRAVIS_BRANCH-$ARCH"
       docker push "$IMG_TAG"
       docker push "$BASE_PUSH_TARGET:devel-$TRAVIS_BRANCH"
       docker push "$BASE_PUSH_TARGET:$BUILD_STAMP"
       docker push "$BASE_PUSH_TARGET:$BUILD_VERSION"
       docker push "$BASE_PUSH_TARGET:$BUILD_VERSION-$BUILD_INFO"
       docker push "$BASE_PUSH_TARGET:latest"
+      docker push "$BASE_PUSH_TARGET:$TRAVIS_BRANCH-$ARCH"
+      docker manifest create "$BASE_PUSH_TARGET:$TRAVIS_BRANCH" "$BASE_PUSH_TARGET:$TRAVIS_BRANCH-amd64" "$BASE_PUSH_TARGET:$TRAVIS_BRANCH-ppc64le"
+      docker manifest push "$BASE_PUSH_TARGET:$TRAVIS_BRANCH"
     fi
-  - make docs
-
+  - if [[ "$ARCH" == "amd64" ]]; then make docs; fi #f5devcentral/attributions-generator:latest docker image available only for x86.
```

Comment:
If F5 has to support this product, then we need to build the docs (and attributions) for all platforms. We can't allow any exceptions.